src/tables/components/table-caption.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+function CaptionComponent({ caption, className }) {
+  return <caption className={className}>{caption}</caption>;
+}
+
+export default CaptionComponent;
```

Comment:
By convention we match the name of the file with the name of the component. E.g., https://github.com/LaunchPadLab/lp-components/blob/master/src/tables/components/table-header.js